docs/usage/config-overview.md

```diff
@@ -4,9 +4,15 @@ When Renovate runs on a repository, the final config used is derived from the:
 
 - Default config
 - Global config
+  - File config
+  - Environment config
+  - CLI config
 - Inherited config
-- Repository config
 - Resolved presets referenced in config
+- Repository config
+
+The above is the order of precedence.
```

Comment:
## Better layout

I think we should:

1. Move the platform sentence about the "order of preference" to the top of the new numbered list.
2. Convert the list to a numbered list.
3. Say something like: "Items with a higher number override (or merge with) items that have lower numbers. If the item has the mergeable property, it will merge with lower numbers."

### Mockup

Platform sentence.

1. Item one
2. Item two
3. Item three, and so on

Explain how _high_ numbers override or merge with _low_ numbers.
Explain the mergeable property.
Explain where you can find the mergeable property.

Reply:
Hi,

I updated the branch. Is this ok for you?